src/test/java/nextstep/theme/ThemeE2ETest.java

```diff
@@ -41,27 +106,53 @@ public void showThemes() {
         assertThat(response.jsonPath().getList(".").size()).isEqualTo(1);
     }
 
-    @DisplayName("테마를 삭제한다")
+    @DisplayName("어드민이 테마를 삭제하면 204 코드 응답")
     @Test
-    void delete() {
-        Long id = createTheme();
+    void deleteThemeByAdminMember() {
+        Long id = createTheme(adminToken);
 
-        var response = RestAssured
+        RestAssured
                 .given().log().all()
-                .when().delete("/themes/" + id)
+                .header(AuthorizationTokenExtractor.AUTHORIZATION, adminToken)
+                .when().delete("/admin/themes/" + id)
                 .then().log().all()
-                .extract();
+                .statusCode(HttpStatus.NO_CONTENT.value());
+    }
 
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+    @DisplayName("어드민이 아닌 멤버가 테마를 삭제하면 403 코드 응답")
+    @Test
+    void deleteThemeByNormalMember() {
+        Long id = createTheme(adminToken);
+
+        RestAssured
+                .given().log().all()
+                .header(AuthorizationTokenExtractor.AUTHORIZATION, normalToken)
+                .when().delete("/admin/themes/" + id)
+                .then().log().all()
+                .statusCode(HttpStatus.FORBIDDEN.value());
+    }
+
+    @DisplayName("로그인 하지 않은 멤버가 테마를 삭제하면 401 코드 응답")
+    @Test
+    void deleteThemeByUnknown() {
+        Long id = createTheme(adminToken);
+
+        RestAssured
+                .given().log().all()
+                .header(AuthorizationTokenExtractor.AUTHORIZATION, invalidToken)
+                .when().delete("/admin/themes/" + id)
+                .then().log().all()
+                .statusCode(HttpStatus.UNAUTHORIZED.value());
     }
 
-    public Long createTheme() {
+    public Long createTheme(String token) {
```

Comment:
마이너하지만 접근 제어자도 신경써주면 좋을 거 같아요~
```suggestion
    private Long createTheme(String token) {
```